tests/test_toast_browser.py

```
@@ -34,6 +38,81 @@ def tearDownClass(cls) -> None:
+        source = (ROOT / "site-dist/components/toast/index.html").read_text(
+            encoding="utf-8"
+        )
+        for match in re.finditer(
+            r'<textarea name="data" hidden>(.*?)</textarea>',
+            source,
+            re.DOTALL,
+        ):
+            payload = json.loads(unescape(match.group(1)).strip())
+            if payload.get("title") == title:
+                return payload
+        raise AssertionError(f"CodePen payload not found: {title}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)